backend/src/main/java/gov/cdc/usds/simplereport/api/accountrequest/AccountRequestController.java

```diff
@@ -68,6 +68,12 @@ private void init() {
   @PostMapping("/waitlist")
   public void submitWaitlistRequest(@Valid @RequestBody WaitlistRequest request)
       throws IOException {
+    Boolean containsFormHoneypot = Boolean.valueOf(request.getFormHoneypot());
+    if (containsFormHoneypot) {
+      log.error(
+          "Waitlist request is probably a bot submission. Aborting waitlist email submission");
```

Comment:
would love any input for whether we want to log anything additionally here.

Reply:
Is there a point where we would want to escalate from logging to alerting?

Reply:
I think this works for now! If we think it's helpful in the future maybe worth sanitizing the input and printing it to see what they fill in. At the moment, I don't see the value in that so I'm ok with this log.

Reply:
Dan mentioned wanting a low-urgency alert to fire during work times [here](https://skylight-hq.slack.com/archives/C064P7MHM47/p1712848872582759) so will figure out how to configure that in Azure in a followup